### PR TITLE
Plot Options Subdialog.  X-Axis and Y-Axis unshown options

### LINKED ISSUE #5555: Plot Options Subdialog bug
**Describe the bug**
When I click on the Plot Options in the Scatter Plot dialog, the Plot Options Subdialog does not show All X-Axis and Y-Axis Options like Scales etc.

@rdstern found this bug.

plot options

### PR DESCRIPTION
**Pull request by Patowhiz**
Fixes #5555
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

**Comment by rdstern** [comment]
@maxwellfundi could you review this and hopefully merge?  Currently (Version 0.5.3) we can't change the x or y axis limits.  The options are not visible.

**Comment by maxwellfundi** [comment]
okay let me have a look at this @rdstern

File: instat/ucrAxes.vb [hunk 1]
```diff
@@ -341,15 +341,15 @@ Public Class ucrAxes
         grpMajorBreaks.Hide()
         grpMinorBreaks.Hide()
         grpScales.Hide()
-        If strAxisType = "Continuous" Then
+        If strAxisType.ToLower = "continuous" Then
             'show continous panels
```
Comment (maxwellfundi) [inline instat/ucrAxes.vb]: @Patowhiz I dont see why you need to have .lower here because the combobox items are all in lower case?

**Comment by maxwellfundi** [comment]
@Ogik99  you were adding the date controls on ucrAxes? How is that going?

File: instat/ucrAxes.vb (hunk 1 above)
Comment (maxwellfundi) [reply instat/ucrAxes.vb]: It does not make much difference!

**Pull request merged**